<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QWidget" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NIM Adjuster</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelNIM">
     <property name="text">
      <string>2205551033</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30pt;</string>
     </property>
    </widget>
   </item>

   <!-- Font Size -->
   <item>
    <layout class="QVBoxLayout" name="fontSizeLayout">
     <item>
      <widget class="QLabel" name="labelFontSize">
       <property name="text">
        <string>Font Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderFontSize">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Font Color -->
   <item>
    <layout class="QVBoxLayout" name="fontColorLayout">
     <item>
      <widget class="QLabel" name="labelFontColor">
       <property name="text">
        <string>Font Color (Grayscale)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderFontColor">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Background Color -->
   <item>
    <layout class="QVBoxLayout" name="bgColorLayout">
     <item>
      <widget class="QLabel" name="labelBGColor">
       <property name="text">
        <string>Background Color (Grayscale)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderBG">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
